cloud/cmd/inter_cluster_gateway/main.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+	"github.com/google/gopacket/routing"
+	"k8s.io/klog/v2"
+)
+
+const (
+	GenevePort    = 6081
+	FirepowerPort = 2615
+)
+
+var (
+	remoteIcgwIPstr   string
+	localDividerIPstr string
+
+	localDividerIP net.IP
+	remoteIcgwIP   net.IP
+	router         routing.Router
+
+	icgwHrdAddr    net.HardwareAddr
+	dividerHrdAddr net.HardwareAddr
+
+	icgwSrc    net.IP
+	dividerSrc net.IP
+
+	handle *pcap.Handle
+	opts   gopacket.SerializeOptions
+	buffer gopacket.SerializeBuffer
+)
+
+// initialize the commandline options
+func InitFlag() {
+	var logLevel string
+	flag.StringVar(&remoteIcgwIPstr, "remote_icgw", "", "the ip address of the remote Inter-Cluster Gateway")
```

Comment:
Shall we add help to show how to use the command with parameters

Reply:
It is given in the PR description.

Reply:
If we close  the PR, how could other people know the usage of the command?

Reply:
I expect a detailed doc after this work is finalized.